librz/debug/dsession.c

```diff
@@ -278,6 +279,7 @@ static void serialize_registers(Sdb *db, HtUP *registers) {
 static bool serialize_memory_cb(void *db, const ut64 k, const void *v) {
 	RzDebugChangeMem *mem;
 	RzVector *vmem = (RzVector *)v;
+	char tmpbuf[16];
```

Comment:
since there is the `0x` this should be at least 19. set this to 32 so it is mem-aligned.

Reply:
Done